apps/picsa-tools/climate-tool/src/app/pages/site-select/site-select.page.ts

```diff
@@ -66,6 +91,9 @@ export class SiteSelectPage {
   }
 
   goToSite(site: IStationMeta) {
+    this.configurationService.updateUserSettings({
+      climate_tool: { station_id: site.id },
+    });
```

Comment:
nit(non-blocking): I think you took the right approach by adding most of the data manipulation methods to the data service (e.g. set/get preferred station ids), but then you have also replicated the same code in the component - better to just call the service code instead